libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/VoiceInstructionsTextPlayer.kt

```diff
@@ -129,6 +140,7 @@ internal class VoiceInstructionsTextPlayer(
     private fun play(announcement: String) {
         val bundle = Bundle().apply {
             putFloat(TextToSpeech.Engine.KEY_PARAM_VOLUME, volumeLevel)
+            putString(TextToSpeech.Engine.KEY_PARAM_STREAM, streamType.toString())
```

Comment:
Is this the same as calling `TextToSpeech#setAudioAttributes()`? See comment above. Should we use `setAudioAttributes` in both `TextToSpeech` and `MediaPlayer` for consistency? 🤔

Reply:
AudioAttributes has different codes for streams and them doesn't match with ours. In GM we can't use it. We can add new method or options, but for our impl we need stream type